resources/views/waitingrooms/index.blade.php

```diff
@@ -9,32 +9,24 @@
     <div class="container">
         <div class="wrapper">
             <div class="container__block">
-                <a class="button" href="{{ route('waitingrooms.create') }}">Add Waiting Room</a>
-            </div>
-        </div>
-    </div>
-
-    <div class="container">
-        <div class="wrapper">
-            <div class="container__block">
                 <div class="table-responsive">
                     <table class="table">
                       <thead>
                         <tr class="table__header">
-                          <th class="table__cell">ID</th>
-                          <th class="table__cell">Campaign</th>
-                          <th class="table__cell">Campaign Run</th>
-                          <th class="table__cell">Start Date</th>
-                          <th class="table__cell">End Date</th>
+                          <th class="table__cell">Waiting Room ID</th>
+                          <th class="table__cell">Contest ID</th>
+                          <th class="table__cell">Signup Start Date</th>
+                          <th class="table__cell">Signup End Date</th>
                           <th class="table__cell">Split</th>
+                          <th class="table__cell">Options</th>
                         </tr>
                       </thead>
                       <tbody>
                         @foreach($rooms as $room)
                           <tr class="table__row">
                             <td class="table__cell"><a href="{{ route('waitingrooms.show', $room->id) }}">{{ $room->id }}</a></td>
-                            <td class="table__cell">{{ $room->campaign_id }}</td>
-                            <td class="table__cell">{{ $room->campaign_run_id }}</td>
+                            {{-- TODO: Link to contest. --}}
+                            <td class="table__cell"><a href="#">{{ $room->contest_id }}</a></td>
                             <td class="table__cell">{{ date('F d, Y', strtotime($room->signup_start_date)) }}</td>
```

Comment:
This should be returned as a Carbon instance from the model (I thiiiiiink) so you should be able to do `$room->signup_start_date->format('F d, Y');` rather than converting it to a string and formatting that.